nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ExecuteProcess.java

```diff
@@ -146,6 +148,13 @@ public class ExecuteProcess extends AbstractProcessor {
       .defaultValue(" ")
       .build();
 
+    static final PropertyDescriptor MIME_TYPE = new PropertyDescriptor.Builder()
+            .name("mime-type")
+            .displayName("Mime Type")
```

Comment:
Recommend naming this `Output MIME Type` for clarity, also recommend using the same value for `name` following the pattern of existing properties in this class:
```suggestion
    static final PropertyDescriptor OUTPUT_MIME_TYPE = new PropertyDescriptor.Builder()
            .name("Output MIME Type")
            .displayName("Output MIME Type")
```